Model/Transaction.php

```diff
@@ -141,6 +142,32 @@ public function getTransactionOrder()
         return $oOrder;
     }
 
+    /**
+     * Returns transaction's payment method name.
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    public function getTransactionPaymentMethodName()
+    {
+        $sPaymentId = $this->getPaymentType();
+        if ($sPaymentId) { // there is an order associated with the transaction
+            $oPayment = PaymentMethodHelper::getPaymentById($sPaymentId);
+            return $oPayment->oxpayments__oxdesc->value;
+        }
+
+        $oXml = simplexml_load_string($this->getResponseXML());
```

Comment:
Do you see this as something needed in the paymentSDK to get the payment method name over a getter in the success response?

Reply:
No. getTransactionPaymentMethodName() is used in admin on Transactions table view, and gets the payment name out of xml which is already stored in database. It's not directly connected to the success response and its getters.

Reply:
Ok.